lib/user_interface.rb

```diff
@@ -0,0 +1,20 @@
+class UserInterface
+  def initialize(formatter:, displayer:, input:)
+    @formatter = formatter
+    @displayer = displayer
+    @input = input
+  end
+
+  def display_board(board_rows)
```

Comment:
Should the `UserInterface` know about a board?

Reply:
We have `board_rows` and `display_board`, but nothing about the rest of the `UserInterface` suggests that it is specific to boards.